pkg/skaffold/initializer/init.go

```diff
@@ -216,9 +217,18 @@ func autoSelectBuilders(builderConfigs []InitBuilder, images []string) ([]builde
 }
 
 func detectBuilders(path string) ([]InitBuilder, error) {
+	// Check for jib
+	if builders := jib.ValidateJibConfigFunc(path); builders != nil {
```

Comment:
Why not return `builders`?

Reply:
`Cannot use builders (type []jib.Jib) as type []InitBuilder in return argument`, had to do the loop as a workaround.

Reply:
oh, ok. Makes sense